### ISSUE TITLE
todos from eric regarding new api playground

### ISSUE THREAD

**Issue by BelfordZ** [MetaMask/metamask-docs#815]
- [x] #816
   - [ ] #817
 - [ ] #818
 - [ ] hotkeys
   - [x] #826
   - [x] #827
 - [x] #819
 - [x] #820
 - [x] #821
 - [x] #822
 - [x] #823
 - [x] #824
 - [x] #825
 - [ ] #830

**Comment by httpJunkie** [comment]
Can the Playground remember the return result from [eth_subscribe](https://docs.metamask.io/wallet/reference/eth_subscribe/) and
ensure it is populated on the [eth_unsubscribe](https://docs.metamask.io/wallet/reference/eth_unsubscribe/) page after a subscribe request?

**Comment by BelfordZ** [comment]
yep! this would be something we could have working with the concept of [links](https://spec.open-rpc.org/#link-object)

If we implement links, in the docs we can remember the results from previous calls, and give suggestions for future params based on what you've seen already. Ty for the suggestion, its a great one!
